website/server/models/challenge.js

```diff
@@ -409,4 +409,12 @@ schema.methods.closeChal = async function closeChal (broken = {}) {
   Promise.all(backgroundTasks);
 };
 
+schema.methods.sanitizeUpdate = function challangeSanitizeUpdate (updateObj) {
+  // _.assign(this, schema.static.sanitizeUpdate(updateObj)); ?
```

Comment:
can this comment be removed?

Reply:
I think `_.assign` - better than `_.merge` in that case. But i don't know app good enough, and `_.assign` may broke something.

Reply:
So it's up to you